src/style.css

```diff
@@ -0,0 +1,57 @@
+@font-face {
+  font-family: Avenir;
+  src: url(../src/fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: bold;
+  src: url(../src/fonts/Avenir-Heavy.ttf);
+}
+
+html {
+  font-family: Avenir, sans-serif;
+}
+
+body {
+  font-weight: 300;
+  padding-top: 12px;
+}
```

Comment:
transfer font-weight to html
and why do you need padding-top?

Reply:
because without padding-top - in npm tests my searchbars is higher

Reply:
this is not the reason to add padding to body
this is a reason to fix your searchbars